server/test/user.route.v1.test.ts

```diff
@@ -18,56 +18,107 @@ describe("User Route v1", () => {
     let user1: SocoboUser = new SocoboUser();
```

Comment:
As stated above I would also prefer to have some unit tests for the single modules (auth, user service ...). These endpoint tests here are very important to test the the functionality as a whole, but they are not well suited to test each component in isolation. See also the comment above.

Since you are testing two things at a time here (the auth and the user server / user routes) you cannot easily locate the error. It could alway be one of these two components. And a change of one of these two components would alway force you to update the tests even if the other component has not changed.